setup.md

```diff
@@ -24,78 +24,144 @@ title: Setup
 
 
 
-> ## Software
-> [Python](http://python.org) is a popular language for
-> scientific computing, and great for general-purpose programming as
-> well.  Installing all of its scientific packages individually can be
-> a bit difficult, so we recommend an all-in-one installer.
+> ## Installing Python using Anaconda
+> [Python][python] is a popular language for scientific computing, and great for
+> general-purpose programming as well. Installing all of its scientific packages
```

Comment:
Do you think we should limit the range of "scientific packages" we talk about here to the ones we use in the lesson?

Reply:
I don't think so. This is just very confusing if you don't really know anything about python. I frequently have this with Ruby which I use very rarely and get a bit overwhelmed with random package names.

Reply:
I meant something like `Installing all of the scientific packages we use in the lesson individually...`